src/api/java/mezz/jei/api/recipe/transfer/IRecipeTransferError.java

```diff
@@ -23,7 +23,14 @@
 		 * Errors that the player can fix. Missing items, inventory full, etc.
 		 * Something informative will be shown to the player.
 		 */
-		USER_FACING
+		USER_FACING,
+
+		/**
+		 * Errors that still allow the usage of the recipe transfer button.
+		 * Hovering over the button will display the error, however the button is active and can be used.
+		 */
```

Comment:
Please bump the JEI version to 6.0.2 and add a line to the docs here:
`* @since JEI version 6.0.2`
This will let users of the API know that they need to require this version if they use this feature.